src/redux/books/books.js

```diff
@@ -35,20 +18,38 @@ const addremoveReducer = (state = data, action = {}) => {
     case REMOVE_BOOK: {
       return [...state.filter((book) => book.id !== action.payload)];
     }
+    case GET_BOOKS: {
+      return action.payload;
+    }
     default:
       return state;
   }
 };
 
 // Action Creators
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
-
-export const removeBook = (payload) => ({
-  type: REMOVE_BOOK,
-  payload,
-});
+
+export const fetchBooks = () => async (dispatch) => {
```

Comment:
- [ ] As the previous reviewer mentioned kindly use `createAsyncThunk` for creating thunk action creators, as per the requirement you should use `fetch` or `Axios` either way you need to use `createAsyncThunk`